### PR TITLE
Associate .xhtml with HTML language mode

### LINKED ISSUE #3521: Syntax highlighting not available for .xhtml files
By default, there appears to be no syntax highlighting for .xhtml files.

### PR DESCRIPTION
**Pull request by Tyriar**
Fixes #3521 

---

Waiting for Feb release to be finalized before pushing

### CODE REVIEW

**Comment by eligrey** [comment]
Thanks for the fix!

File: extensions/html/package.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
 	"contributes": {
 		"languages": [{
 			"id": "html",
-			"extensions": [ ".html", ".htm", ".shtml", ".mdoc", ".jsp", ".asp", ".aspx", ".jshtm" ],
+			"extensions": [ ".html", ".htm", ".shtml", ".xhtml", ".mdoc", ".jsp", ".asp", ".aspx", ".jshtm" ],
 			"aliases": [ "HTML", "htm", "html", "xhtml" ],
 			"mimetypes": ["text/html", "text/x-jshtm", "text/template", "text/ng-template"]
```
Comment (eligrey) [inline extensions/html/package.json]: Oh you should also probably add `application/xhtml+xml` to this list.

Comment (Tyriar) [inline extensions/html/package.json]: Good call, will do

Comment (Tyriar) [inline extensions/html/package.json]: https://github.com/Microsoft/vscode/commit/2212fbbb6ce19688e5cb7da5952d5f9dc8066f49

**Pull request merged**